plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/TestFullParquetReader.java

```diff
@@ -13,11 +13,24 @@
  */
 package io.trino.plugin.hive.parquet;
 
+import org.testng.annotations.Test;
+
+// Failing on multiple threads because of org.apache.hadoop.hive.ql.io.parquet.write.ParquetRecordWriterWrapper
+// used during test initialization
```

Comment:
`used` -> `is being used`.

Add comment what's wrong with `ParquetRecordWriterWrapper`

Reply:
`AbstractTestParquetReader` has `tester` field, but `ParquetTester` looks immutable.
Do you know where is the mutable state held?

Reply:
What do you mean by mutable state? There is no shared state between those tests. Everything on our side is kept in local variables

Reply:
So why singlethreaded-ness helps, if there is no shared state?

Reply:
because of this:
```
// Failing on multiple threads because of org.apache.hadoop.hive.ql.io.parquet.write.ParquetRecordWriterWrapper
// uses a single record writer across all threads.
```
?

Reply:
That's why i am asking about the shared mutable state.
Is ParquetRecordWriterWrapper instance shared? where?
or, does ParquetRecordWriterWrapper use a shared _static_ instance?

sorry for bothering you, i wish i know that and it seems you know all that already